### DISCUSSION TITLE
Project build failing on node-gyp install

### DISCUSSION THREAD

**Discussion by reed2ep** [microsoft/vscode-discussions#1151] [Extension Development QnA]
Been having issues trying to debug vscode to verify some changes i made. I followed the setup steps on both my laptop and pc, however, i get this same error on both. This leads me to believe im either missing some step, or im downloading some dependency incorrectly. I tried the following from the setup guide: "run git clean -xfd in your vscode folder, then re-run yarn." but still have issues. I havent been able to figure out the exact area within the preinstall script that is having issue. Ill admit im new to this sort of development, so some oversight on my end is likely here. I appreciate any help in advance.

![image](https://github.com/microsoft/vscode-discussions/assets/71101339/c196d3bc-0c6d-40e8-a7b2-f5ca02715824)

**Comment by soulshined** [comment]
@reed2ep  can you elaborate on why this was closed as resolved? did you find a solution or did you just close it to close it?
